src/components/Pages/Unit/widgets/Downloads/MediaDownloads.js

```diff
@@ -197,10 +198,15 @@ class MediaDownloads extends Component {
   };
 
   renderRow = (file, label, t) => {
+    const { chronicles }     = this.props;
```

Comment:
What do you think about supplying callback here instead of both `chronicles` and `onClick`?
And if not callback then parameter to renderRow?

Reply:
Fixed for callback.